azerothcore/azerothcore-wotlk
# fix(DB/smart_scripts): Fix Quest: The Restless Souls

## Changes Proposed:
-  Fix links in smart scripts 
-  

## Issues Addressed:

- Closes [Quest : [The Restless Souls] #3194](https://github.com/chromiecraft/chromiecraft/issues/3194)

## SOURCE:


## Tests Performed:

- Used Egan's Blaster on Restless Soul now gives kill credit.


## How to Test the Changes:


1.  Accept the Quest "[The Restless Souls](https://wowgaming.altervista.org/aowow/?quest=5282)" (part 2, the Egan's Blaster one)
2. Enter Stratholme and use Egan's Blaster on Ghostly Citizen and/or Spectral Citizen
3. Use Egan's Blaster on the spawned Restless Soul
4. Should give kill credit.

## Known Issues and TODO List:


- [Freed Soul(11136 in creature_template) should say one random phrase from database. It's not happening. All entries are in db but script is not triggering.  ]


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
